user-manual/transfer/import-metadata.rst

```diff
@@ -345,8 +345,44 @@ metadata in Archivematica <premis-template>`.
    :file: _csv/rights.csv
    :header-rows: 1
 
-The ``rights.csv`` file is parsed by the job "Load Rights" within the 
-"Characterize and Extract Metadata" microservice run during 
+The ``rights.csv`` file is parsed by the job "Load Rights" within the
+"Characterize and Extract Metadata" microservice run during
 :ref:`transfer <transfer>`.
 
+Importing structural metadata with mets_structmap.xml
```

Comment:
You need a header - this should solve the build error that Travis is displaying.

```suggestion
.. _structmap.xml::

Importing structural metadata with mets_structmap.xml
```